dynamic/clientset/dynamic_clientset.go

```diff
@@ -105,74 +99,57 @@ func (rc *ResourceClient) Kind() string {
 }
 
 func (rc *ResourceClient) GroupVersion() schema.GroupVersion {
-	return rc.gv
+	return rc.resource.GroupVersion()
 }
 
 func (rc *ResourceClient) GroupResource() schema.GroupResource {
```

Comment:
`GroupResource()` is still necessary for `ResourceClient` because it is called [here](https://github.com/GoogleCloudPlatform/metacontroller/blob/master/dynamic/informer/informer.go#L125).